docs/deployment.md

```diff
@@ -10,44 +10,110 @@ This documentation assumes that you have a working LogSearch deployment and that
 
-#### UAA OAuth
+#### 4-a) Standalone Kibana with UAA OAuth enabled
 
-Logsearch-for-CloudFoundry provides [kibana plugin](https://github.com/logsearch/logsearch-for-cloudfoundry/tree/develop/src/kibana-cf_authentication) to ask the user for credentials and perform an OAuth handshake with the CloudFoundry UAA server before serving requests.
+Logsearch-for-CloudFoundry provides [kibana plugin](https://github.com/logsearch/logsearch-for-cloudfoundry/tree/develop/src/kibana-cf_authentication) to ask the user for credentials and perform an OAuth handshake with the CloudFoundry UAA server before serving requests. The plugin can be added to the standalone Kibana deployed as part of the Logsearch release. If you are choosing to proceed with a standalone Kibana and enable the authentication in it, then use `templates/logsearch-for-cf.example-with-uaa-auth.yml` stub and customise it with your deploy settings:
 
 ```sh
 $ vim templates/logsearch-for-cf.example-with-uaa-auth.yml
 $ scripts/generate_deployment_manifest ~/workspace/logsearch.yml templates/logsearch-for-cf.example-with-uaa-auth.yml > ~/workspace/logsearch-with-logsearch-for-cf.yml
 ```
 
-### 4. Update the logsearch deployment with the new manifest
+#### 4-b) Kibana deployed as CF application with UAA OAuth enabled
+
+There is a possibility to deploy [Kibana as a CloudFoundry application](features.md#possibility-to-deploy-kibana-as-cloudfoundry-application). Deployed Kibana will already include the authentication plugin providing UAA OAuth. To enable this feature use `templates/stub.logsearch-for-cloudfoundry.yml` stub and customise it with your deploy settings.
+
+> NOTE: If you choose to deploy Kibana as a CF application, then, most probably, you don't need to have a standalone Kibana instance anymore (one deployed in Logsearch release). You can disable it in `~/workspace/logsearch.yml`:
+> ```yml
+> jobs:
+> ...
+> - instances: 0
+>   name: kibana
+>   ...
+> ```
+
+```sh
+$ vim templates/stub.logsearch-for-cloudfoundry.yml
+$ scripts/generate_deployment_manifest ~/workspace/logsearch.yml templates/stub.logsearch-for-cloudfoundry.yml > ~/workspace/logsearch-with-logsearch-for-cf.yml
+```
+
+### 5. Update the logsearch deployment with the new manifest
 
 ```sh
 $ bosh deployment ~/workspace/logsearch-with-logsearch-for-cf.yml
 $ bosh deploy
 ```
 
-### 5. Update Cloud Foundry deployment to forward component logs to ingestor
+#### 5-a) If deploy Standalone Kibana with UAA OAuth enabled
 
-```yaml
-properties:
-  syslog_daemon_config:
-    address: haproxy-static-ip
-    port: 5514
+If you have chosen to deploy a standalone Kibana with UAA authentication enabled, then you should additionally run `create-uaa-client` errand task after the deploy:
+
+```sh
+$ bosh run errand create-uaa-client
 ```
+> NOTE: Before running the job, make sure to create a security group in your CF which you pass as `cf-kibana.cloudfoundry.api_security_group`:
```

Comment:
In case of standalone VM with Kibana we can make calls to Cloud Foundry API without CF security groups configured. This part must be included in 5b.